### DISCUSSION TITLE
how to remove lock screen button from start menu

### DISCUSSION THREAD

**Discussion by Chrysanthemum536** [linuxmint/discussions#427] [Questions]
how to remove lock screen button from start menu

**Comment by Hilyxx** [comment]
You can remove sessions buttons with  - right click (on the applet) --> configure -  but you can't remove only lock screen button.

**Comment by claudiux** [comment]
[#7592](https://github.com/linuxmint/Cinnamon/issues/7592)

**Comment by billyswong** [comment]
If you are interested in hiding the button only and not concerned about how to disable this function, see https://forums.linuxmint.com/viewtopic.php?t=410798
